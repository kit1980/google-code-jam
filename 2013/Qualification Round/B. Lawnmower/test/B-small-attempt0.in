100
3 3
2 1 2
1 1 1
2 1 2
5 5
2 2 2 2 2
2 1 1 1 2
2 1 2 1 2
2 1 1 1 2
2 2 2 2 2
9 7
1 2 1 1 2 2 2
2 2 2 1 2 1 1
2 2 2 2 2 2 2
1 2 2 2 2 2 2
1 2 2 1 1 1 1
2 1 2 2 2 2 2
2 2 2 1 1 2 1
1 1 1 1 1 2 1
1 2 2 2 2 2 2
5 1
1
1
1
1
1
10 10
1 1 1 1 1 1 1 1 1 2
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 2 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
8 7
2 2 2 2 2 1 2
2 2 2 2 2 1 2
1 1 1 1 1 1 1
1 1 1 1 1 1 1
1 1 1 1 1 1 1
2 2 2 2 2 1 2
1 1 1 1 1 1 1
2 2 2 2 2 1 2
4 1
1
1
1
1
10 10
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 2 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 2 1 1
3 2
1 1
1 1
1 1
5 3
1 1 1
1 1 1
2 2 2
1 1 1
1 2 2
2 4
2 2 1 1
1 1 1 1
7 9
1 2 1 1 1 2 1 2 2
2 2 1 2 2 2 2 2 2
1 1 2 1 2 2 2 1 2
1 1 2 2 1 1 1 2 2
2 2 1 2 1 2 2 1 1
1 2 1 1 2 1 1 1 1
1 1 2 2 2 1 2 1 1
8 7
1 1 1 2 1 2 2
2 2 2 1 1 2 1
2 1 2 1 2 1 2
1 1 1 1 2 1 1
1 2 2 1 2 1 1
2 2 1 2 2 2 1
1 2 2 2 2 2 2
1 2 2 2 2 1 1
10 10
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 2 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
10 10
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 1 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
3 2
1 2
2 2
2 1
9 5
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
1 1 1 1 1
8 7
2 1 2 1 1 1 2
1 1 1 1 1 1 1
2 1 2 1 1 1 2
2 1 2 1 1 1 2
1 1 1 1 1 1 1
1 1 1 1 1 1 1
2 1 1 1 1 1 2
1 1 1 1 1 1 1
2 3
1 1 2
2 2 1
5 3
1 2 2
2 2 2
2 2 1
1 2 2
1 2 2
10 10
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 1 2 2 2 2 2
9 9
1 1 1 1 1 2 1 2 2
2 2 1 1 2 1 2 2 2
2 2 2 1 1 1 2 1 1
2 2 1 1 1 1 1 1 2
2 1 2 1 2 1 2 2 1
1 1 2 2 2 1 1 1 1
2 2 1 2 2 2 1 1 2
2 1 2 1 2 2 2 2 2
1 2 2 1 2 1 1 2 2
6 9
2 2 1 1 1 1 1 2 1
1 1 1 1 1 1 1 1 1
2 2 1 1 1 1 1 2 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
10 10
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
2 6
1 1 1 1 1 1
1 1 1 2 1 2
1 2
1 1
4 6
1 1 1 1 1 1
1 1 1 1 1 1
1 1 1 1 1 1
1 1 1 1 1 1
2 1
1
2
2 6
2 1 2 2 2 2
2 2 2 2 2 2
9 3
2 2 2
2 1 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 1
8 3
1 1 2
1 1 2
1 1 2
1 1 1
1 1 1
1 1 1
1 1 1
1 1 1
1 1
2
2 5
1 1 1 1 1
1 2 1 1 1
5 2
2 1
1 1
1 2
2 2
2 2
5 7
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
1 2 2 2 2 2 2
2 2 2 2 2 2 2
9 3
1 1 1
2 2 1
2 2 1
2 2 1
2 2 1
2 2 1
1 1 1
2 2 1
2 2 1
7 4
1 1 1 1
1 1 1 1
1 2 2 2
1 2 2 2
1 1 1 1
1 2 2 2
1 1 1 1
10 10
2 2 2 2 1 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 1
2 2 2 2 2 2 2 2 2 2
10 8
2 2 2 2 2 2 1 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2
1 4
1 1 1 1
2 4
2 2 1 2
2 2 2 2
10 10
2 1 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
5 3
1 2 2
1 1 1
1 2 2
1 2 2
1 2 2
4 2
1 1
2 1
1 2
2 1
9 6
1 1 1 1 1 2
2 2 1 1 1 2
2 1 1 2 1 1
2 1 2 1 2 1
1 1 2 1 1 2
2 2 1 1 2 2
1 2 2 2 2 2
1 2 1 1 1 2
2 2 1 1 2 1
7 8
1 2 2 2 1 1 1 1
1 1 1 1 1 1 1 1
1 2 2 2 1 1 1 2
1 1 1 1 1 1 1 1
1 2 2 2 1 1 1 2
1 1 1 1 1 1 1 1
1 2 2 2 1 1 1 2
3 2
2 1
2 1
1 2
2 5
2 2 2 2 2
2 2 2 2 1
10 10
2 1 2 2 2 2 1 2 1 2
2 2 2 2 1 2 2 2 1 2
1 1 2 2 2 2 2 2 2 2
2 2 1 2 2 2 2 2 2 2
2 2 2 1 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 1 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
4 6
1 2 2 2 2 2
2 2 2 2 2 2
2 2 2 2 2 2
2 2 2 2 2 2
5 7
2 1 2 1 2 2 1
1 2 2 1 2 1 2
1 1 2 2 1 1 1
1 2 2 2 2 1 2
2 2 1 2 2 2 1
6 9
1 1 1 1 2 1 2 1 1
1 1 1 1 2 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 2 1 2 1 1
1 1 1 1 2 1 2 1 1
4 7
2 2 2 1 2 2 1
1 2 1 2 2 2 1
2 2 1 1 2 2 2
2 1 2 1 1 1 2
7 4
2 2 2 2
2 1 2 2
1 2 2 1
1 2 1 2
1 2 2 1
2 2 2 2
1 1 1 2
9 7
1 1 1 2 1 1 2
1 1 1 2 1 1 2
1 1 1 2 1 1 2
1 1 1 2 1 1 2
1 1 1 1 1 1 1
1 1 1 1 1 1 1
1 1 1 1 1 1 1
1 1 1 1 1 1 1
1 1 1 1 1 1 1
7 8
2 2 2 2 1 2 2 2
2 2 2 2 2 2 2 2
2 2 1 2 2 2 2 1
2 1 2 2 2 1 2 2
2 2 2 2 2 2 2 1
2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 1
6 2
1 1
2 1
1 1
1 2
2 1
1 2
7 8
1 2 1 1 2 1 2 1
2 2 1 2 2 2 2 1
2 2 2 1 2 2 2 1
1 1 2 2 2 2 2 2
2 2 2 2 1 1 2 1
1 1 2 1 1 1 2 1
2 1 2 2 1 2 2 1
2 3
2 2 2
1 2 2
10 8
1 1 1 1 1 1 1 1
1 1 2 1 1 2 1 2
1 1 1 1 1 1 1 1
1 1 2 1 1 2 1 2
1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1
8 3
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 1
10 10
1 1 2 2 2 1 2 1 2 1
1 1 2 2 1 2 2 2 2 2
1 2 1 2 2 1 2 1 2 1
2 2 2 2 1 1 1 2 2 2
1 2 2 1 2 2 1 2 2 1
1 2 2 1 1 1 2 2 1 1
1 1 2 2 2 1 1 1 2 2
2 2 1 1 2 1 2 1 2 1
2 1 1 2 1 1 1 2 1 1
1 1 2 2 2 1 1 1 2 2
6 4
2 1 1 2
1 1 1 1
1 2 2 2
1 1 2 2
1 1 1 2
2 2 1 2
2 3
2 2 1
1 2 2
2 3
1 1 2
1 1 2
2 1
2
2
1 4
2 1 1 2
10 10
2 2 2 2 2 2 2 2 2 2
2 1 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
2 2 2 1 2 2 2 2 2 2
2 2 2 2 2 2 2 2 2 2
1 2
1 1
10 10
1 1 1 2 1 2 2 1 1 1
1 1 1 2 1 2 2 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 2 1 2 2 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 2 1 2 2 1 1 1
1 1 1 2 1 2 2 1 1 1
1 1 1 2 1 2 2 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 2 1 2 2 1 1 1
3 2
2 2
2 2
2 1
2 4
2 2 2 2
2 2 2 1
8 7
2 2 2 1 2 2 2
1 2 2 2 2 1 2
2 2 2 2 2 1 2
2 2 2 2 1 2 2
2 2 2 2 1 2 2
1 2 2 2 2 1 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
3 9
2 1 2 2 2 1 1 1 1
1 2 1 1 1 2 1 1 2
2 2 1 1 2 2 1 2 2
4 1
1
1
1
1
2 6
2 2 2 2 2 1
2 2 2 2 2 2
9 3
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 2 2
2 1 2
9 9
1 2 2 2 1 1 1 1 1
1 1 2 2 1 1 1 1 1
1 2 2 2 1 1 1 1 1
1 2 2 2 1 1 1 1 1
1 2 2 2 1 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
1 2 2 2 1 1 1 1 1
1 1 1 1 1 1 1 1 1
10 10
1 2 2 2 2 2 1 1 1 1
1 2 2 2 2 2 1 1 1 1
1 2 2 2 2 2 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 2 2 2 2 2 1 1 1 1
1 2 2 2 2 2 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
9 5
2 2 2 2 2
2 2 2 2 2
2 2 2 2 2
2 2 2 2 2
2 2 2 2 2
1 2 2 2 2
2 2 2 2 2
2 2 2 2 2
2 2 2 2 2
3 8
1 2 2 1 1 1 1 1
1 2 2 1 1 1 2 2
2 2 1 1 1 2 1 1
4 6
2 2 2 2 2 2
2 2 2 2 2 2
2 2 2 2 2 2
2 2 2 1 2 2
7 4
2 2 2 2
2 2 2 2
2 2 2 2
2 2 2 1
2 2 2 2
2 2 2 2
2 2 2 2
5 7
1 1 1 1 1 1 1
2 1 2 1 1 1 1
1 1 1 1 1 1 1
2 1 2 1 1 1 1
1 1 1 1 1 1 1
3 5
1 2 1 1 2
2 1 2 2 2
1 1 1 2 2
7 8
1 1 1 2 1 2 1 2
1 1 1 1 1 1 1 1
1 1 1 2 1 2 1 2
1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1
1 1 1 2 1 2 1 2
1 1 1 1 1 1 1 1
1 5
1 2 1 1 2
10 10
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
1 1 1 1 2 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1
7 5
1 2 1 2 1
1 1 2 2 2
2 1 1 2 1
2 2 2 1 1
1 2 2 1 2
2 2 1 2 2
1 1 2 1 1
9 5
1 1 2 2 2
1 2 2 2 2
1 2 2 1 1
2 2 2 2 1
2 1 2 2 1
2 2 2 2 2
2 1 2 1 2
2 2 2 2 1
2 2 1 1 2
9 9
1 2 2 1 2 1 2 1 2
2 1 2 1 1 2 1 1 2
2 2 1 1 2 2 2 1 1
2 2 1 2 2 2 2 1 1
2 1 1 2 1 2 1 1 1
2 2 2 1 2 2 1 1 2
2 2 2 2 2 1 2 2 1
2 2 1 2 2 1 1 1 1
2 2 2 1 2 1 2 1 2
8 3
2 1 2
1 2 2
1 2 2
2 2 1
2 1 1
2 2 2
2 2 1
2 1 1
9 9
2 1 2 1 2 1 1 1 1
1 1 1 1 1 1 1 1 1
2 1 2 1 2 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
2 1 2 1 2 1 1 1 1
1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1
9 7
2 2 2 2 2 2 2
1 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 2 2 2 2 2 2
2 5
1 1 2 1 1
2 2 2 2 2
1 1
1
6 9
2 1 2 2 2 1 2 2 1
2 2 2 2 2 2 2 1 2
1 2 2 2 1 2 2 1 2
2 2 1 2 2 2 1 1 1
2 2 2 2 2 2 2 2 2
2 1 2 2 2 2 1 2 2
10 10
1 2 1 1 1 1 2 1 1 2
1 2 2 1 2 1 1 2 1 1
2 2 2 2 2 2 1 1 2 1
1 1 2 2 2 2 2 1 2 1
2 2 2 1 2 2 2 1 1 2
2 2 2 2 2 1 1 2 1 2
1 2 1 2 1 2 1 2 1 1
1 2 2 1 1 1 1 2 2 2
2 2 2 2 1 1 1 1 1 1
1 1 2 1 2 1 2 2 1 2
8 10
1 2 1 2 2 2 1 1 2 1
1 1 2 2 2 1 2 2 1 1
2 2 1 2 2 1 1 2 2 1
2 1 2 2 2 2 1 1 2 2
2 1 1 1 1 2 2 2 2 1
1 2 2 2 2 2 2 2 1 2
2 2 1 1 1 2 1 2 1 1
2 2 2 1 2 2 1 1 2 1
10 8
2 2 2 2 2 1 2 1
1 1 1 1 1 1 1 1
2 2 2 2 2 1 2 1
2 2 2 2 2 1 2 1
2 2 2 2 2 1 2 1
1 2 2 2 2 1 2 1
2 1 2 2 2 1 2 1
2 2 2 2 2 1 2 1
2 2 2 2 2 1 2 1
2 2 2 2 2 1 2 1
