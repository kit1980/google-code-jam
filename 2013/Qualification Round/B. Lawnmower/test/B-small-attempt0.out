Case #1: YES
Case #2: NO
Case #3: NO
Case #4: YES
Case #5: NO
Case #6: YES
Case #7: YES
Case #8: NO
Case #9: YES
Case #10: NO
Case #11: YES
Case #12: NO
Case #13: NO
Case #14: YES
Case #15: NO
Case #16: NO
Case #17: YES
Case #18: NO
Case #19: NO
Case #20: NO
Case #21: NO
Case #22: NO
Case #23: YES
Case #24: YES
Case #25: YES
Case #26: YES
Case #27: YES
Case #28: YES
Case #29: NO
Case #30: NO
Case #31: YES
Case #32: YES
Case #33: YES
Case #34: NO
Case #35: NO
Case #36: YES
Case #37: YES
Case #38: NO
Case #39: NO
Case #40: YES
Case #41: NO
Case #42: NO
Case #43: YES
Case #44: NO
Case #45: NO
Case #46: NO
Case #47: NO
Case #48: NO
Case #49: NO
Case #50: NO
Case #51: NO
Case #52: NO
Case #53: NO
Case #54: NO
Case #55: YES
Case #56: NO
Case #57: NO
Case #58: NO
Case #59: NO
Case #60: YES
Case #61: NO
Case #62: NO
Case #63: NO
Case #64: NO
Case #65: YES
Case #66: YES
Case #67: YES
Case #68: NO
Case #69: YES
Case #70: YES
Case #71: NO
Case #72: NO
Case #73: NO
Case #74: NO
Case #75: YES
Case #76: NO
Case #77: NO
Case #78: NO
Case #79: YES
Case #80: NO
Case #81: NO
Case #82: NO
Case #83: NO
Case #84: YES
Case #85: NO
Case #86: YES
Case #87: YES
Case #88: YES
Case #89: NO
Case #90: NO
Case #91: NO
Case #92: NO
Case #93: YES
Case #94: NO
Case #95: NO
Case #96: YES
Case #97: NO
Case #98: NO
Case #99: NO
Case #100: NO
